console/console-init/ui/src/modules/address/dialogs/CreateAddress/CreateAddress.tsx

```diff
@@ -139,6 +139,11 @@ export const CreateAddress: React.FunctionComponent = () => {
           if (deadletterAddress && deadletterAddress.trim() !== "") {
             variable.spec.deadletter = deadletterAddress.trim();
           }
+        }
+        if (
```

Comment:
It can be handle in single `if` check.
`
if ((addressType?.trim().toLowerCase() === "queue" ||
          addressType.trim().toLowerCase() === "topic") && expiryAddress?.trim()!=="" 
  ) {
          variable.spec.expiry = expiryAddress.trim();
        }
`